src/icons/HiddenEyeIcon.tsx

```diff
@@ -7,7 +7,7 @@ export interface Props {
   color?: Colors
 }
 
-function HiddenEyeIcon({ color = Colors.contentPrimary, size }: Props) {
+function HiddenEyeIcon({ color = Colors.accent, size = 24 }: Props) {
```

Comment:
i removed the default props in a previous PR and the size broke, so adding it back